scripts/upload-resources-to-github

```diff
@@ -67,7 +67,7 @@ upload_asset() {
     # HTTP success code expected - 201 Created
     if [[ $response_code -eq 201 ]]; then
         asset_id=$(echo $response_content | jq '.id')
-        ASSET_IDS_UPLOADED+=($asset_id)
+        ASSET_IDS_UPLOADED+=("$asset_id")
```

Comment:
I don't think line 78 will work since it's just saving the string to the ASSETS variable?